charts/fluent-bit/values.yaml

```diff
@@ -9,8 +9,9 @@ replicaCount: 1
 image:
   repository: cr.fluentbit.io/fluent/fluent-bit
   # Overrides the image tag whose default is {{ .Chart.AppVersion }}
-  tag: ""  # Set to "-" to not use the default value
-  digest: ""
+  # Set to "-" to not use the default value
```

Comment:
What does this mean? We set the `tag: -`?

Reply:
It mean you can use a digest without a tag by setting `tag` to `-`; this is already in the chart. A use case would be using a specific version of the public Chainguard image which only has the `latest` & `latest-dev` tags.

FYI last time I checked the Chainguard Fluent Bit image was v2.1.2 instead of v2.1.4 (latest at the time), any ideas why?

Reply:
Ah right, could you tweak the comment slightly then to clarify? I don't like having to delve into the details to figure it out.

They have probably not rebuilt it I imagine is why. I'm not sure how they pick up upstream changes.

Reply:
This is existing code and as I said I'm focusing on the new charts, if I was to fix the docs for this chart I'd be here forever. I almost didn't port this code over for this reason.

RE Chainguard could it be because the release notes are out of date and the site is saying that v2.1.2 is the latest version?

Reply:
Ah yeah, although there are release notes (well announcements) for 2.1.3.

CI auto-creates the Github release for everything though so we have that for all of them and probably should be the source of truth. I don't know anyway, more a question for the Chainguard folks.